lasagna-master/lasagna-master.js

```diff
@@ -0,0 +1,58 @@
+/// <reference path="./global.d.ts" />
+
+import { createUnzip } from "zlib"
+
+// @ts-check
+
+/**
+ * Implement the functions needed to solve the exercise here.
+ * Do not forget to export them so they are available for the
+ * tests. Here an example of the syntax as reminder:
+ *
+ * export function yourFunction(...) {
+ *   ...
+ * }
+ */
+export function cookingStatus(time) {
```

Comment:
Once you add the airbnb config properly to the .eslintrc file, you should update this function accordingly.